.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  extends: "plugin:jest-dom/recommended",
+  parser: "@babel/eslint-parser",
+  rules: {
+    // your configuration
+  },
+};
```

Comment:
- Kindly enable your linters.

Reply:
Thanks for pointing it out. I have already included those rules in .eslintrc.js. 
Also, I have a question. Shouldn't the request be taken as invalid because eslint was not right configured?
However, I have already implemented it and it works right.
Thanks for your time.